packages/mermaid/src/docs/community/n00b-overview.md

```diff
@@ -63,6 +63,6 @@ In fact one can pick up the syntax for it quite easily from the examples given a
 
 ## Mermaid is for everyone.
 
-Video [Tutorials](https://mermaid-js.github.io/mermaid/#/../config/Tutorials) are also available for the mermaid [live editor](https://mermaid.live/).
+Video [Tutorials](https://mermaid-js.github.io/mermaid/#/./Tutorials) are also available for the mermaid [live editor](https://mermaid.live/).
```

Comment:
This is the old link. You should use https://mermaid.js.org/config/Tutorials.html instead.

Reply:
`https://mermaid-js.github.io/mermaid/#/../config/Tutorials` does not work like in the docs and `https://mermaid-js.github.io/mermaid/#/./Tutorials` already redirects to `https://mermaid.js.org/config/Tutorials.html`.